packages/realm/rollup.config.mjs

```diff
@@ -64,7 +68,8 @@ export default [
     },
     plugins: [
       nodeResolve({
-        resolveOnly: ["path-browserify"],
+        mainFields: ["react-native", "module", "main"],
```

Comment:
Ensuring the `react-native` main field is used when importing the `@realm/network-transport` package.